package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-railio-form-components",
-  "version": "0.0.11",
+  "version": "0.0.12",
```

Comment:
Next time. you don't need to update the version or add a tag on every project. It will be done after merging :-)